### DISCUSSION TITLE
Build Error: Unknown flag --build-arg

### DISCUSSION THREAD

**Discussion by SworxOnThings** [moby/moby#46393] [General]
I am getting this build error "unknow flag --build-arg" when trying the command "make BUILD_DIR=. shell". 
Can someone help here?

**Comment by tonystz** [comment]
I encounter the same issue

**Reply by thaJeztah** [reply]
@tonystz what version of docker are you running? `--build-arg` has been supported for a long time; if you get that error, it may be that you're running a really old version of docker.
